Resources/Prototypes/Nyanotrasen/Recipes/Cooking/recipes_moth.yml

```diff
@@ -0,0 +1,496 @@
+
+- type: microwaveMealRecipe
+  id: RecipeMothHerbyCheese
+  name: herby cheese recipe
+  result: FoodMothHerbyCheese
+  time: 10
+  solids:
+    FoodCurdCheese: 1
+    FoodAmbrosiaVulgaris: 1 #Herbs
+
+- type: microwaveMealRecipe
+  id: RecipeMothMothicSalad
+  name: mothic salad recipe
+  result: FoodMothSaladBase
+  time: 10
+  solids:
+    FoodCabbage: 1
+    FoodOnionRedSlice: 2
+    FoodTomato: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothPesto
+  name: pesto recipe
+  result: FoodMothPesto
+  time: 5
+  reagents:
+    OilOlive: 5
+    TableSalt: 5
+  solids:
+    FoodGarlic: 1
+    FoodAmbrosiaVulgaris: 1 #Herbs
+
+- type: microwaveMealRecipe
+  id: RecipeMothBechamel
+  name: bechamel recipe
+  result: FoodMothBechamel
+  time: 5
+  reagents:
+    Milk: 10
+    Flour: 5
+  solids:
+    FoodButter: 1
+
+# Entrees
+
+- type: microwaveMealRecipe
+  id: RecipeMothSqueakingFry
+  name: squeaking stir fry recipe
+  result: FoodMothSqueakingFry
+  time: 15
+  reagents:
+    OilOlive: 5
+  solids:
+    FoodTofuSlice: 2
+    FoodOnionSlice: 1
+    FoodChili: 1
+    FoodCarrot: 1
+    FoodCheeseCurds: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothToastedSeeds
+  name: toasted seeds recipe
+  result: FoodMothToastedSeeds
+  time: 15
+  reagents:
+    OilOlive: 5
+  solids:
+    WheatSeeds: 1
+    CarrotSeeds: 1
+    PoppySeeds: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothChiliCabbageWrap
+  name: sweet chili cabbage wrap recipe
+  result: FoodMothChiliCabbageWrap
+  time: 15
+  reagents:
+    Sugar: 5 #Honey
+  solids:
+    FoodCheeseSlice: 1 #Grilled cheese slice
+    FoodMothSaladBase: 1
+    FoodChili: 1
+    FoodCabbage: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothBakedCheese
+  name: baked cheese recipe
+  result: FoodMothBakedCheese
+  time: 20
+  solids:
+    FoodCheese: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothBakedCheesePlatter
+  name: baked cheese platter recipe
+  result: FoodMothBakedCheesePlatter
+  time: 5
+  solids:
+    FoodMothBakedCheese: 1
+    FoodBreadPlainSlice: 3 #3 toasts
+
+- type: microwaveMealRecipe
+  id: RecipeMothBakedRice
+  name: baked rice recipe
+  result: FoodMothBakedRice
+  time: 15
+  solids:
+    FoodRiceBoiled: 1
+    FoodPotato: 2
+    FoodCabbage: 1
+    FoodChili: 1
+    FoodAmbrosiaVulgaris: 1 #Herbs
+
+# Meals
+
+- type: microwaveMealRecipe
+  id: RecipeMothGreenLasagne
+  name: green lasagne recipe
+  result: FoodMothGreenLasagne
+  time: 20
+  solids:
+    FoodNoodlesBoiled: 1
+    FoodCheeseSlice: 1
+    FoodMothBechamel: 1
+    FoodMothPesto: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothBakedCorn
+  name: baked corn recipe
+  result: FoodMothBakedCorn
+  time: 15
+  solids:
+    FoodCorn: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothButteredBakedCorn
+  name: buttered baked corn recipe
+  result: FoodMothButteredBakedCorn
+  time: 5
+  solids:
+    FoodMothBakedCorn: 1
+    FoodButter: 1
```

Comment:
Making a note that one of my refactor targets is using molten butter in places, and this is a prime example; A whole stick for one ear of corn could use a pass of common sense.